pkgs/tools/inputmethods/ibus/default.nix

```diff
@@ -47,6 +48,14 @@ let
       makeWrapper ${glib.dev}/bin/glib-mkenums $out/bin/glib-mkenums --unset PYTHONPATH
     '';
   };
+  # make-dconf-override-db.sh needs to execute dbus-launch in the sandbox,
```

Comment:
What error does this cause?

Reply:
```
dbus[19730]: Failed to start message bus: Failed to open "/etc/dbus-1/session.conf": No such file or directory
```

I wrote it in the comment.

Reply:
Oh, right. I did not notice we were previously replacing the file with a blank script.

Reply:
Can confirm this works to copy the default value from GSettings schemas to the dconf database. Though I do not understand why would they do it, unless they do not use GSettings and access dconf database directly.